src/Npgsql/NpgsqlTypes/NpgsqlRange.cs

```diff
@@ -256,8 +256,8 @@ static bool IsEmptyRange([CanBeNull] T lowerBound, [CanBeNull] T upperBound, Ran
             if (!HasEquatableBounds)
                 return lowerBound?.Equals(upperBound) ?? false;
 
-            IEquatable<T> lower = (IEquatable<T>)lowerBound;
-            IEquatable<T> upper = (IEquatable<T>)upperBound;
+            var lower = (IEquatable<T>)lowerBound;
+            var upper = (IEquatable<T>)upperBound;
```

Comment:
Move it to an additional PR?

Reply:
I think it's small enough to leave in...